link/xdp_test.go

```diff
@@ -0,0 +1,47 @@
+package link
+
+import (
+	"testing"
+
+	"github.com/cilium/ebpf"
+	"github.com/cilium/ebpf/asm"
+	"github.com/cilium/ebpf/internal/testutils"
+)
+
+const IfIndexLO = 1
+
+func TestAttachXDP(t *testing.T) {
+	testutils.SkipOnOldKernel(t, "5.7", "BPF_LINK_TYPE_XDP")
```

Comment:
This appeared in 5.9 I think https://github.com/torvalds/linux/commit/aa8d3a716b59db6c1ad6c68fb8aa05e31980da60

Reply:
Updated.